tests/drillhole_v4_0_test.py

```diff
@@ -911,9 +912,17 @@ def test_add_data_to_property(tmp_path):
             "interval_values_a", spatial_index=True
         )
 
+        verification_map_value = (verification["interval_values_a"] * 10).astype(
+            np.int32
+        )
+        verification_map_value -= np.min(verification_map_value)
+
+        value_map = {idx: f"{idx}" for idx in np.unique(verification_map_value)}

```

Comment:
This map would actually not be valid for GA since it always needs the `0: "Unknown"` entry at the beginning. Maybe it's a good time to add a validation on the `value_map` setter.

Reply:
Shouldn't we check keys and add an "unknown"?
Or just check if present?

Reply:
Improved setter method for value map.